protein/src/org/labkey/protein/ProteinContainerListener.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.labkey.protein;
+
+import org.jetbrains.annotations.NotNull;
+import org.labkey.api.data.Container;
+import org.labkey.api.data.ContainerManager.ContainerListener;
+import org.labkey.api.security.User;
+
+import java.beans.PropertyChangeEvent;
+import java.util.Collection;
+import java.util.Collections;
+
+public class ProteinContainerListener implements ContainerListener
```

Comment:
This looks like a bunch of no-ops. Is it needed?

Reply:
Protein schema deletes will be moved here once this module manages the `prot` schema. Added a comment.